Dockerfile

```diff
@@ -48,7 +48,7 @@ RUN curl -sL https://storage.googleapis.com/kubernetes-release/release/$( \
     pip install codespell
 
 RUN python3 -m pip install awscli
-RUN curl -sL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.52.2
+RUN curl -sL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.50.1
```

Comment:
This can't be downgraded, older releases of golangci-lint will consume essentially infinite memory when used with golan 1.20.